### PR TITLE
README: document release procedure for new contributors

### PR DESCRIPTION
**Pull request by tlvu**
We are having more new contributors and I want to avoid repeating myself.

### CODE REVIEW

File: birdhouse/README.rst [hunk 1]
```diff
@@ -257,6 +257,48 @@ to ``bump2version`` to auto-generated the tag in other to apply it directly on `
 after the merge, instead of onto the commits in the PR prior merging.
 
 
+Release Procedure
+-----------------
+
+* Update CHANGES.md_, commit, push.
```
[the remark this answers is not in the document]
Comment (tlvu) [reply birdhouse/README.rst:266]: Resolved in 81a0d6179679f6834701e94ed7473a56858313fb

File: birdhouse/README.rst [hunk 2]
```diff
@@ -257,6 +257,48 @@ to ``bump2version`` to auto-generated the tag in other to apply it directly on `
...
+  opened this PR :)
+
+* Only when you are ready to merge the PR immediately, you can continue with
+  the following steps to.  Doing the following steps too early and you might
+  lose the "push race" if someone else is also trying to release at the same
+  time.  Also, in the spirit of not losing the "push race", execute all these
+  steps together, do not take a break in the middle.
+
+  * Merge with ``master`` branch, if needed, so next ``bump2version`` step will
+    bump to the proper next version.
```
[the remark this answers is not in the document]
Comment (tlvu) [reply birdhouse/README.rst]: Resolved in 81a0d6179679f6834701e94ed7473a56858313fb

File: birdhouse/README.rst [hunk 3]
```diff
@@ -257,6 +257,48 @@ to ``bump2version`` to auto-generated the tag in other to apply it directly on `
...
+    policy" section above.  Push.
+
+  * Merge this PR, copying the entire PR description into the merge commit
+    description.  This is so that the page
+    https://github.com/bird-house/birdhouse-deploy/tags will contain relevant
+    info nicely.  That page was previously used as an ad-hoc changelog before
+    CHANGES.md_ was formally introduced.
+
+  * Run ``git tag`` on the commit created the by merge, with the same tag as
+    ``bump2version`` generated.
```
[the remark this answers is not in the document]
Comment (tlvu) [reply birdhouse/README.rst:303]: Resolved in 81a0d6179679f6834701e94ed7473a56858313fb

**Comment by tlvu** [comment]
FYI for readme update like this that do not impact the deployment, I won't cut a release.

File: birdhouse/README.rst [hunk 4]
```diff
@@ -257,6 +257,55 @@ to ``bump2version`` to auto-generated the tag in other to apply it directly on `
 after the merge, instead of onto the commits in the PR prior merging.
 
 
+Release Procedure
+-----------------
+
+* Pull/merge latest ``master`` to make sure modifications applied in
```
Comment (fmigneault) [inline birdhouse/README.rst:263]: missing "[are] applied in ..."

**Review by fmigneault — APPROVED** [review]
Small wording to adjust, then good to merge.

File: birdhouse/README.rst (hunk 4 above)
Comment (tlvu) [reply birdhouse/README.rst]: 2c7e21370c6097630b936c635e5458f796a7f559

**Pull request merged**